test/open_mic_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/open_mic'
+require './lib/joke'
+require './lib/user'
+
+class OpenMicTest < Minitest::Test
+  def setup
+    @open_mic = OpenMic.new({location: "Comedy Works", date: "11-20-18"})
+    @sal = User.new("Sal")
+    @ali = User.new("Ali")
+    @joke1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
+    @joke2 = Joke.new(2, "How do you keep a lion from charging?", "Take away its credit cards.")
+  end
+
+  def test_it_exists
+    assert_instance_of OpenMic, @open_mic
+  end
+
+  def test_it_has_attributes
+    assert_equal "Comedy Works", @open_mic.location
+  end
+
+  def test_it_has_performers
+    assert_equal [], @open_mic.performers
+  end
```

Comment:
great job up to here. You can combine these last two tests into a `test_has_attributes` method to check anything you initialized